bin/post_process_sce.R

```diff
@@ -81,75 +106,89 @@ if(all(is.na(sce$miQC_pass))){
   metadata(sce)$scpca_filter_method <- "miQC"
 }
 
+# include ADT filtering information
+colData(sce)$scpca_filter[adt_discard_rows] <- "Remove"
```

Comment:
very minor, but I don't think we need the `colData()` here: we call it later as just `sce$scpca_filter`, so I think we should be able to be consistent? (Also modify existing usage above)

```suggestion
sce$scpca_filter[adt_discard_rows] <- "Remove"
```